sparta/CMakeLists.txt

```diff
@@ -109,6 +109,10 @@ if (NOT rc EQUAL "0")
   message (FATAL_ERROR "could not run git command 'git describe --tags --always', rc=${rc}")
 endif ()
 
+if (USE_LTO)
+  set (LTO_FLAGS "-flto -Ofast")
```

Comment:
-Ofast enables -ffast-math enables -funsafe-math-optimizations and  -ffinite-math-only, which can cause problems if you use NaN or inf...
also can use add_compile_options() here as well to avoid having to set a variable

Reply:
Yep, we encountered one issue with the fast-math -- log2 utils in sparta.  See issue #112.  We decided to go ahead with this enhancement as it bought about 10%+